src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -471,10 +470,13 @@
           short-summary: The node taints for the node pool. You can't change the node taints through CLI after the node pool is created.
         - name: --priority
           type: string
-          short-summary: The priority of the node pool. Regular or Low.
+          short-summary: The priority of the node pool. Regular or Spot.
```

Comment:
Advice: You can use get_enum_type. Allowed values can showed automatically in help. And framework will add checks of validity of values.
Example:
c.argument('os_type', arg_type=get_enum_type(['Windows', 'Linux'])

Reply:
semantically it may be cleaner. but it reduced our flexibility to add other validation in the future.

Reply:
updated.